src/core/mac/sub_mac.hpp

```diff
@@ -623,6 +626,8 @@ class SubMac : public InstanceLocator, private NonCopyable
     void StartTimerForBackoff(uint8_t aBackoffExponent);
     void BeginTransmit(void);
     void SampleRssi(void);
+    void TimerStart(uint32_t aDelayUs);
+    void TimerStartAt(Time aStartTime, uint32_t aDelayUs);
```

Comment:
How about `StartTimer()` and `StartTimerAt()`? (help make the name similar to `StartTimerForBackoff()`?